resources/user.rb

```diff
@@ -37,11 +38,16 @@ def load_current_value
 end
 
 action :create do
+  package 'openssl' do
+    action :nothing
+  end.run_action(:install)
+
   user new_resource.name do
-    password new_resource.password
+    password generate_system_password
```

Comment:
Looks like you don't use the `password` property anymore
Probably should do `new_resource.password || generate_system_password ` instead so that it is still possible to specify a password